pom.xml

```diff
@@ -169,6 +198,28 @@
                     <plugin>
                         <groupId>org.codehaus.mojo</groupId>
                         <artifactId>rpm-maven-plugin</artifactId>
+                        <executions>
```

Comment:
Executions are inherited from the parent. You don't need to configure them again.

Reply:
done